docs/introduction-to-smart-contracts.rst

```diff
@@ -505,12 +505,19 @@ Delegatecall / Callcode and Libraries
 There exists a special variant of a message call, named **delegatecall**
 which is identical to a message call apart from the fact that
 the code at the target address is executed in the context of the calling
```

Comment:
I think the most important aspect of "context" here is `address`. If you split up the sentence here this sounds like you are explaining what "context" means.